src/pages/proposal-commons/proposal-details/proposal-details.tsx

```diff
@@ -158,6 +159,15 @@ const ProposalDetailsContent = (props: ProposalDetailsProps) => {
         </div>
       </div>
 
+      {isMalicious && (
+        <div className={styles.malicious}>
+          <WarningIcon aria-hidden />
+          <p>
+            <b>This proposal is potentially malicious.</b> A suspicious EVM script has been detected.
```

Comment:
Maybe we could mention that the proposal has not been created via the UI.

Reply:
I think it's fine as is 👍 If you feel strongly about it, then perhaps rope Burak in (he was happy with the current message btw)

Reply:
No, I also think the current message is fine. It was just a suggestion.